README.md

```diff
@@ -147,7 +147,7 @@ npm run test-with-coverage
 
 ```json
 "scripts": {
-  "test-with-coverage": "jest --coverage && cat ./coverage/lcov.info | codacy-coverage"
+  "test-with-coverage": "jest --coverage || cat ./coverage/lcov.info | codacy-coverage"
```

Comment:
This makes no sense. `||` will make the second step only run when the first fails, which makes no sense.

Also you are not defining a threshold so it should not fail since the flag only fails for a threshold.